### DISCUSSION TITLE
Turbolinks on Cart and Checkout page

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#112] [1. Help]
Hi,

Breeze is disabled in Cart and checkout page. I want to use **Turbolinks.visit('checkout/cart')** on the Breeze disabled (Cart/Checkout) pages. 

I tried it by adding in my custom Ajax event but it is not working. Can you please guide me on this ?

Thanks

**Comment by vovayatsyuk** [comment]
Sorry, but you have to implement it on your own. I never tried this and don't know how to do that.
